### PR TITLE
virt-manager: fix build on darwin

### PR DESCRIPTION
**Pull request by wegank**
###### Description of changes

Supersedes #152350 (and #184925 for gtk-vnc).

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/applications/virtualization/virt-manager/default.nix [hunk 1]
```diff
@@ -64,6 +64,10 @@ python3.pkgs.buildPythonApplication rec {
     gappsWrapperArgs+=(--prefix PATH : "${lib.makeBinPath [ cpio e2fsprogs file findutils gzip ]}")
 
     makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+
+    # Fixes testCLI0051virt_install_initrd_inject on Darwin: "cpio: root:root: invalid group"
+    substituteInPlace virtinst/install/installerinject.py \
+      --replace "'--owner=root:root'" "'--owner=0:0'"
```
Comment (bjornfor) [inline pkgs/applications/virtualization/virt-manager/default.nix:70]: Sounds like this patch should be sent upstream.

**Review by bjornfor — APPROVED** [review]

File: pkgs/applications/virtualization/virt-manager/default.nix (hunk 1 above)
Comment (wegank) [reply pkgs/applications/virtualization/virt-manager/default.nix:70]: virt-manager/virt-manager#423.

Comment (bjornfor) [reply pkgs/applications/virtualization/virt-manager/default.nix:70]: @wegank: Thank you!

**Comment by mikroskeem** [comment]
Thought coming back to work on my old PR and saw you having already solved this. Thank you for getting virt-manager up and running 🙏

**Pull request merged**